diff --git a/feeds/luci/collections/luci/Makefile b/feeds/luci/collections/luci/Makefile
index 9b495c3..d34f50f 100644
--- a/feeds/luci/collections/luci/Makefile
+++ b/feeds/luci/collections/luci/Makefile
@@ -11,8 +11,8 @@ LUCI_BASENAME:=luci
 
 LUCI_TITLE:=Standard OpenWrt set including full admin with ppp support and the default Bootstrap theme
 LUCI_DEPENDS:= \
-	+uhttpd +uhttpd-mod-ubus +luci-mod-admin-full +luci-theme-bootstrap \
-	+luci-app-firewall +luci-proto-ppp +libiwinfo-lua +IPV6:luci-proto-ipv6 \
+	+uhttpd +uhttpd-mod-ubus +luci-mod-admin-full \
+	+libiwinfo-lua +IPV6:luci-proto-ipv6 \
 	+rpcd-mod-rrdns
 
 PKG_LICENSE:=Apache-2.0
